godot-macros/src/class/data_models/field_export.rs

```diff
@@ -277,10 +280,21 @@ impl FieldExport {
             quote! { None }
         };
 
-        let mut options = HashSet::new();
+        let mut flags = HashSet::<String>::new();
+        let mut kvs = HashMap::<String, String>::new();
 
-        while let Some(option) = parser.next_allowed_ident(&ALLOWED_OPTIONS[..])? {
-            options.insert(option.to_string());
+        loop {
+            let key_maybe_value =
+                parser.next_allowed_key_optional_value(&FLAG_OPTIONS, &KV_OPTIONS)?;
+            match key_maybe_value {
+                Some((option, None)) => {
+                    flags.insert(option.to_string());
+                }
+                Some((option, Some(value))) => {
+                    kvs.insert(option.to_string(), value.expr()?.to_string());
```

Comment:
One question I had, how do you turn value, which is the expression `"px"` into just that string?  As-is, I'm creating `"\"px\""`, which works for Godot, but it's not done intentionally and I don't like that.

Reply:
Solved with https://github.com/godot-rust/gdext/pull/783#discussion_r1658144236